packages/path_provider/path_provider/example/integration_test/path_provider_test.dart

```diff
@@ -117,6 +117,8 @@ void _verifySampleFile(Directory? directory, String name) {
 
   file.writeAsStringSync('Hello world!');
   expect(file.readAsStringSync(), 'Hello world!');
-  expect(directory.listSync(), isNotEmpty);
+  // This check intentionally avoids using Directory.listSync due to
```

Comment:
For the app-facing package we should probably make this codepath Android-only, and have other platforms keep using `listSync`.

Reply:
Changed!